ElectronicObserver/ViewModels/Translations/FormMainTranslationViewModel.cs

```diff
@@ -73,6 +73,7 @@ public class FormMainTranslationViewModel : TranslationBaseViewModel
 	public string StripMenu_Tool_ConstructionRecord => Properties.Window.FormMain.Tool_ConstructionRecord.Replace("_", "__").Replace("&", "_");
 	public string StripMenu_Tool_ResourceChart => Properties.Window.FormMain.Tool_ResourceChart.Replace("_", "__").Replace("&", "_");
 	public string SenkaViewer => Window.Tools.SenkaViewer.SenkaViewer.Title;
+	public string ExpeditionRecordViewer => "Expedition Record";
```

Comment:
Make a resx file in the expedition record viewer folder and move this text in there, I can make the Japanese translation.